vmdk_plugin/E2E_Tests/e2e_verifyVolmProperties_test.go

```diff
@@ -0,0 +1,177 @@
+		fmt.Println("docker cli check finished - current time: ", time.Now())
+		for i := 0; i < len(volSizes); i++ {
+			for k := 0; k < len(formatTypes); k++ {
+				fmt.Println("Creating a volume of Format Type - ", formatTypes[k])
+				volName := TestInputParamsUtil.GetVolumeNameWithTimeStamp("dockerVol")
+				fmt.Println("Creating a volume - current time: ", time.Now())
+				_, err := TestUtil.InvokeCommand(vms[vmIndx], "docker volume create --driver=vmdk --name="+volName+" -o size="+volSizes[i]+" -o diskformat="+formatTypes[k])
+				if err != nil {
+					fmt.Println("Failed to create a volume named: ", volName)
+					log.Fatal(err)
+				} else {
+					fmt.Println("\n successfully created a  volume -- ", volName)
+					dockerVolmRmvCmd = dockerVolmRmvCmd + " " + volName
+				}
+				fmt.Println("Volume creation finished - current time: ", time.Now())
+				// verifying the disk format using docker cli and admin cli
+				fmt.Println("Getting the disk format from Admin cli - current time: ", time.Now())
+				diskFrmtFrmAdminCli = TestUtil.GetDiskFormatUsingAdminCli(volName, esx)
+				if diskFrmtFrmAdminCli != formatTypes[k] {
+					log.Fatal("Disk format field for volume from Admin cli is ", diskFrmtFrmAdminCli)
+				}
+				fmt.Println("Finished verifying the disk format from Admin cli - current time: ", time.Now())
+				// docker cli check only for docker 1.12 and above
+				if dockerCliCheck {
+					// getting disk format using docker cli
+					fmt.Println("Getting the disk format from Docker cli - current time: ", time.Now())
+					diskFrmtFrmDockerCli = TestUtil.GetDiskFormatUsingDockerCli(volName, vms[vmIndx])
+					fmt.Println("Finished fetching the disk format from Docker cli - current time: ", time.Now())
+					if diskFrmtFrmDockerCli != formatTypes[k] {
+						log.Fatal("TEST - Disk format field for volume from Docker cli is ", diskFrmtFrmDockerCli)
+					}
+					if diskFrmtFrmDockerCli != diskFrmtFrmAdminCli {
+						log.Fatal("Disk format field for volume from docker cli is not same as that from admin cli")
+					}
+					fmt.Println("Getting the capacity from Docker cli - current time: ", time.Now())
+					capacityForVolDockerCli = TestUtil.GetVolSizeUsingDockerCli(volName, vms[vmIndx])
+					fmt.Println("Finished getting the capacity from Docker cli - current time: ", time.Now())
+					if capacityForVolDockerCli != expctdSizeDkerCli {
+						log.Fatal("Size for volume is not " + volSizes[i])
+					}
+				}
+				fmt.Println("Getting the disk size from Admin cli - current time: ", time.Now())
+				// get capacity of volume using Admin Cli
+				capacityForVolAdminCli := TestUtil.GetVolSizeByAdminCli(volName, esx)
+				if capacityForVolAdminCli != expctdSizeAdminCli {
+					log.Fatal("Admin CLI Size for volume is not " + volSizes[i])
+				}
+				fmt.Println("Finished verifying the disk size from Admin cli - current time: ", time.Now())
+
+				fmt.Println("Invoking cmd to run container - current time: ", time.Now())
```

Comment:
A generic question to all fmt.Println() - would it be better to put this in a log ? This way we can review the info long after the test has failed :-). Or do we campture fmt,Printl in some persistent log ?